src/components/Personalization/extractPageWideScopeDecisions.js

```diff
@@ -0,0 +1,25 @@
+import PAGE_WIDE_SCOPE from "./constants/scope";
+
+const isPageWideScope = decision => decision.scope === PAGE_WIDE_SCOPE;
+
+const extractDecisions = decisions => {
+  const pageWideDecisions = [];
+  const viewDecisions = {};
+
+  decisions.forEach(decision => {
+    if (isPageWideScope(decision)) {
+      pageWideDecisions.push(decision);
+    } else {
+      if (!viewDecisions[decision.scope]) {
+        viewDecisions[decision.scope] = [];
+      }
+      viewDecisions[decision.scope].push(decision);
+    }
+  });
+
+  return [pageWideDecisions, viewDecisions];
```

Comment:
Are we returning an array instead of an object in attempt to optimize? It seems like an object would have been the default choice.